contrib/minimize-emulation.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# pylint: disable=invalid-name,missing-docstring
+
+import json
+import sys
+import hashlib
+import zipfile
+
+
+def _minimize_usb_events(data) -> None:
+    for device in data["UsbDevices"]:
+        try:
+            for event in device.get("UsbEvents", []) + device.get("Events", []):
+                event["Id"] = hashlib.sha1(event["Id"].encode()).hexdigest()[:8]
```

Comment:
if you follow my sentinel suggestion don't forget to update here too.

Reply:
Done!